apps/desktop-wallet/src/api/apiDataHooks/useToken.ts

```diff
@@ -78,7 +57,7 @@ const useToken = (id: TokenId): Token => {
       : unlistedFT
         ? (unlistedFT as UnlistedFT)
         : nft || ({ id } as NonStandardToken),
```

Comment:
Same answer as above!

Reply:
So here, if I remove all type assertions, the return value of `useToken` looks like this. It's impossible to tell what it returns (notice the `...` at the end)

<img width="557" alt="image" src="https://github.com/user-attachments/assets/23f8a60c-b898-4c6f-8867-d22cec742867">

Not sure what to do

Reply:
What we could do  (which we are already doing btw here) is this:

```ts
type Token = {
  data: ListedFT | UnlistedFT | NFT | NonStandardToken
  isLoading: boolean
}

const useToken = (id: TokenId): Token => {
```

I like that in the places where `useToken` is used I see the full list of available token options:

<img width="725" alt="image" src="https://github.com/user-attachments/assets/431a6565-91d0-4e6b-9323-975d6a59e1c3">


even if in the hook itself I only see `Token` when I hover over the hook name, but then again the definition is right above anyway so it doesn't matter

<img width="527" alt="image" src="https://github.com/user-attachments/assets/914e036b-0397-4150-af27-6369ea0fcd36">

I'd go for this approach instead of type assertion.

Reply:
Implemented in 1c490834c